### DISCUSSION TITLE
Outflowing waves boundary condition

### DISCUSSION THREAD

**Discussion by nirmal129** [lululxvi/deepxde#1180] [Q&A] [answered]
Hello, 

I am working on the following PDE. 

$$-\frac{\partial^2 \psi}{\partial t^2} + \frac{\partial^2 \psi}{\partial r^2} - V(r)\psi = 0 $$

And the boundary conditions are as follows. 

Left boundary: $$-\frac{\partial \psi}{\partial t}+\frac{\partial \psi}{\partial r}=0$$
Right boundary: $$\frac{\partial \psi}{\partial t}+\frac{\partial \psi}{\partial r}=0$$

Note: at boundaries $V(r)\approx0$, the original PDE reduces to a wave equation, and outflow wave boundary conditions are applied. 

My question is how to implement such boundary conditions using DeepXDE. I found nothing like this in the documentation or discussion.

**Comment by lululxvi** [comment, marked as answer]
Use `OperatorBC` https://deepxde.readthedocs.io/en/latest/modules/deepxde.icbc.html#deepxde.icbc.boundary_conditions.OperatorBC
